[2023-11-25T00:06:10.457+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.Extract scheduled__2023-11-24T00:00:00+00:00 [queued]>
[2023-11-25T00:06:10.464+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.Extract scheduled__2023-11-24T00:00:00+00:00 [queued]>
[2023-11-25T00:06:10.464+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-25T00:06:10.476+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Extract> on 2023-11-24 00:00:00+00:00
[2023-11-25T00:06:10.517+0000] {standard_task_runner.py:57} INFO - Started process 35631 to run task
[2023-11-25T00:06:10.521+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'Extract', 'scheduled__2023-11-24T00:00:00+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpadfoz1gi']
[2023-11-25T00:06:10.526+0000] {standard_task_runner.py:85} INFO - Job 203: Subtask Extract
[2023-11-25T00:06:10.571+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.Extract scheduled__2023-11-24T00:00:00+00:00 [running]> on host e647c96745b0
[2023-11-25T00:06:10.658+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='Extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-24T00:00:00+00:00'
[2023-11-25T00:22:50.193+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-11-25T00:22:50.218+0000] {process_utils.py:131} INFO - Sending 15 to group 35631. PIDs of all processes in the group: [35631]
[2023-11-25T00:22:50.218+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 35631
[2023-11-25T00:22:50.221+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-11-25T00:22:50.312+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=35631, status='terminated', exitcode=0, started='00:06:10') (35631) terminated with exit code 0
